modules/ROOT/pages/installing-and-configuring-runtime-manager-agent.adoc

```diff
@@ -9,1064 +9,403 @@ _Enterprise Edition_
-|*US*|*exchange2-asset-manager-kprod.s3.amazonaws.com* |  443
-|*EU*|*eu1.anypoint.mulesoft.com* | 443
-|*EU*|*mule-manager.eu1.anypoint.mulesoft.com* | 443
-|*EU*|*runtime-manager.eu1.anypoint.mulesoft.com* | 443
-|*EU*|*analytics-ingest.eu1.anypoint.mulesoft.com* |  443
-|*EU*|*arm-auth-proxy.prod-eu.msap.io* |  443
-|*EU*|*data-authenticator.eu1.anypoint.mulesoft.com* |  443
-|*EU*|*exchange2-asset-manager-kprod-eu.s3.eu-central-1.amazonaws.com* |  443
-|===
-
-*Static IPs*
-
-There are two static IPs that must be whitelisted to access the mule-manager hosts.
-
-[%header,cols="3*a"]
-|===
-|Region |Name |IP Address
-|*US*|*mule-manager.anypoint.mulesoft.com* |52.201.174.72
-|*US*|*mule-manager.anypoint.mulesoft.com* |52.201.67.218
-|*EU*|*mule-manager.eu1.anypoint.mulesoft.com* |18.195.19.18
-|*EU*|*mule-manager.eu1.anypoint.mulesoft.com* |18.194.245.32
-|===
-
-The following DNS endpoints do not have static IP addresses:
-
-[%header,cols="2*a"]
-|====
-|Region | Name
-|*US*|*runtime-manager.anypoint.mulesoft.com*
-|*EU*|*runtime-manager.eu1.anypoint.mulesoft.com*
-|====
-
-*Dynamic IPs*
-
-Some of the IP addresses used by Anypoint Platform services are assigned automatically by the underlying cloud infrastructure, and hence we can't guarantee that they are not going to change in the future.
-
-For this reason, you should not implement a whitelist based on the specific IP addresses being assigned to Anypoint services.
+|Parameter|Value| Description
+|`-A`
 
-Nowadays, many firewall devices allow you to define Layer 7 Firewall Rules, where you could be able to filter by destination name or application type.
+`--amc-host`
+| `_amc-host_`
 
-The hostnames that you should include in your Layer 7 Firewall rules include:
+|Specifies the service URL location of your local instance of Runtime Manager, such as `+https://10.0.0.1:8080/hybrid/v1+`.
 
-[%header,cols="2*a"]
-|===
-|Region |Hostname
-|*US*|*anypoint.mulesoft.com*
-|*US*|*analytics-ingest.anypoint.mulesoft.com*
-|*US*|*arm-auth-proxy.prod.cloudhub.io*
-|*US*|*data-authenticator.anypoint.mulesoft.com*
-|*EU*|*eu1.anypoint.mulesoft.com*
-|*EU*|*analytics-ingest.eu1.anypoint.mulesoft.com*
-|*EU*|*arm-auth-proxy.prod-eu.msap.io*
-|*EU*|*data-authenticator.eu1.anypoint.mulesoft.com*
+You can test whether the service is available at `_AMC_HOST_/hybrid/v1`.
+|`-W`
 
-|===
+`--mcm-host`
+| `_mcm-host`
+|Specifies the service URL location of your local instance of MCM, for example, `wss://10.0.0.2:443/mule`.
 
-== Setting up a Proxy
+You can test whether the service is available at `_MCM_HOST_/mule`.
 
-You can configure the Runtime Manager agent to send websocket messages through an HTTP proxy.
+|`-C`
 
-The only proxy authentication method supported by the Runtime Manager agent is Basic Authentication.
+`--cs-host`
+|`_core-services-host_`
+|Specifies the service URL of your local instance of Access Management, for example, `+https://10.0.0.3:8080/accounts+`.
```

Comment:
GLOBALLY rearrange "the service URL location" to "the service location URL". I'll stop marking.